From 1a6fb3b97f452dc53a703222579f1e71a32e33be Mon Sep 17 00:00:00 2001
From: Jingoo Han <jg1.han@samsung.com>
Date: Wed, 26 Feb 2014 10:30:14 +0900
Subject: [PATCH 373/399] spi: fsl-dspi: Use SIMPLE_DEV_PM_OPS macro

commit ba811addff3d29d1ea9861dbfc06e8ef80714f94 upstream

Use SIMPLE_DEV_PM_OPS macro in order to make the code simpler.

Signed-off-by: Jingoo Han <jg1.han@samsung.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
---
 drivers/spi/spi-fsl-dspi.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/spi/spi-fsl-dspi.c b/drivers/spi/spi-fsl-dspi.c
index b113b89..2249cb7 100644
--- a/drivers/spi/spi-fsl-dspi.c
+++ b/drivers/spi/spi-fsl-dspi.c
@@ -436,9 +436,7 @@ static int dspi_resume(struct device *dev)
 }
 #endif /* CONFIG_PM_SLEEP */
 
-static const struct dev_pm_ops dspi_pm = {
-	SET_SYSTEM_SLEEP_PM_OPS(dspi_suspend, dspi_resume)
-};
+static SIMPLE_DEV_PM_OPS(dspi_pm, dspi_suspend, dspi_resume);
 
 static struct regmap_config dspi_regmap_config = {
 	.reg_bits = 32,
-- 
1.7.5.4

